### DISCUSSION TITLE
jhipster 8.4.0 entity User not generated

### DISCUSSION THREAD

**Discussion by djibythiaw** [jhipster/generator-jhipster#26224] [Q&A] [answered]
Hello,
I am using the version 8.4.0 of Jhipster. I have generated a microservice with OAuth2 but I see that the entity User and Authority are not generated. The previous version of jhipster that I used in my last project was the version 7.9.3.
Why this entities are no more generated ? How to generate them in the microservice ?
Thanks

**Comment by mraible** [comment, marked as answer]
I believe this question was answered by @mshima yesterday. https://github.com/jhipster/generator-jhipster/issues/26206#issuecomment-2123077549

**Comment by djibythiaw** [comment]
OK Thank you. It's working fine with the option --sync-user-with-idp when generating the project.
